Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/ShowMoreButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useCallback, useState} from 'react';
+import {LinkButton} from "./Button";
+import {MdExpandLess, MdExpandMore} from "react-icons/md";
+
+import styles from './ShowMoreButton.module.scss'
+interface IShowMoreButtonProps {
+    onClick: () => void;
+}
+
+const ShowMoreButton = ({
+    onClick,
+} : IShowMoreButtonProps) => {
+    const [ showMore, setShowMore] = useState(false);
+    
+    const buttonText = showMore
+        ? "Show less"
+        : "Show more";
+    
+    const buttonIcon = showMore
+        ? <MdExpandLess/>
+        : <MdExpandMore/>;
+    
+    const onShowMoreClick = useCallback(
+        () => {
+            setShowMore(!showMore);
+            onClick();
+        },
+        [showMore],
+    );
+
+    return (
+        <div className={styles.container}>
+            <LinkButton
+              text={buttonText}
+              to={'#'}
+              type={'link'} onClick={onShowMoreClick}
```

Comment:
Split on two lines.
Also, if you do not use the `event` object, just to `onClick={() => onShowMoreClick()}...`